### PR TITLE
VolInfo

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

VolInfo simple tool to get info (size and values) from a vol file

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: volumetric/volInfo.cpp [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+ *
+ *
+ * This file is part of the DGtal library.
+ */
+
+#include <iostream>
+#include <map>
+#include <DGtal/base/Common.h>
+#include <DGtal/io/readers/VolReader.h>
+#include <DGtal/io/writers/VolWriter.h>
```
Comment (kerautret) [inline volumetric/volInfo.cpp:31]:
```suggestion
```
not used

File: volumetric/volInfo.cpp [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ * @date 2021/02/11
+ *
+ *
+ * This file is part of the DGtal library.
+ */
+
+#include <iostream>
+#include <map>
+#include <DGtal/base/Common.h>
+#include <DGtal/io/readers/VolReader.h>
```
Comment (kerautret) [inline volumetric/volInfo.cpp:30]:
```suggestion
#include <DGtal/io/readers/GenericReader.h>
```
to handle ITK images ?

File: volumetric/volInfo.cpp [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  app.add_option("-i,--input,1", inputFileName, "Input vol file." )->required()->check(CLI::ExistingFile);
+  
+  app.get_formatter()->column_width(40);
+  CLI11_PARSE(app, argc, argv);
+  // END parse command line using CLI ----------------------------------------------
+
+  trace.beginBlock("Loading file");
+  typedef ImageContainerBySTLVector<Z3i::Domain, unsigned char>  MyImageC;
+
+  MyImageC  imageC = VolReader< MyImageC >::importVol ( inputFileName );
```
Comment (kerautret) [inline volumetric/volInfo.cpp:110]: If yes for GenericReader :
```suggestion
  MyImageC  imageC = GenericReader<MyImageC>::import(inputFileName);
```

**Review by kerautret — COMMENTED** [review]
Thanks for the new tools just some typo and proposition to use GenericReader

**Comment by dcoeurjo** [comment]
thx for the feedbacks. Fixed

**Comment by kerautret** [comment]
> thx for the feedbacks. Fixed

you are welcome, you just forget the duplicate doc first remarks

File: volumetric/volInfo.cpp [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+
+ @b Usage: 	./volumetric/volInfo [OPTIONS] 1 [2]
+
+
+ @b Allowed @b options @b are : 
+ @code
+ Positionals:
+   1 TEXT:FILE REQUIRED                  Input vol file.
+
```
Comment (kerautret) [inline volumetric/volInfo.cpp:55]:
```suggestion
 ```
already

File: volumetric/volInfo.cpp [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+#include "CLI11.hpp"
+
+
+using namespace std;
+using namespace DGtal;
+using namespace Z3i;
+
+/**
+ @page volSubSample volSubSample
```
Comment (kerautret) [inline volumetric/volInfo.cpp:43]:
```suggestion
 @page volInfo volInfo
```

**Comment by kerautret** [comment]
@dcoeurjo if you review PR #433 it will fix the CI check ;) and I suggested other small typo that I miss

**Review by kerautret — APPROVED** [review]

**Comment by dcoeurjo** [comment]
👍

**Pull request merged**